Lib/idlelib/calltips.py

```diff
@@ -123,6 +123,8 @@ def get_entity(expression):
 _INDENT = ' '*4  # for wrapped signatures
 _first_param = re.compile(r'(?<=\()\w*\,?\s*')
 _default_callable_argspec = "See source or doc"
+_invalid_method = "invalid method signature"
+_argument_positional = "('/' marks preceding arguments as positional-only)"
```

Comment:
Upon seeing a real example, I don't like the clash between () as call operator and comment marker in tips such as

(object, /)
('/' marks preceding arguments as positional-only)
Append object to the end of the list.

I prefer either [] or # as the comment marker.  What think you?

(object, /)
['/' marks preceding arguments as positional-only]
Append object to the end of the list.

(object, /)
# '/' marks preceding arguments as positional-only
Append object to the end of the list.